recipes/data-management/sth/ha/deploy_back.sh

```diff
@@ -2,6 +2,6 @@
 set -o allexport
 . ./backend.env
 set +o allexport
-# docker network create --opt com.docker.network.driver.mtu=${DOCKER_MTU:-1500} -d overlay backend
-curl -O https://raw.githubusercontent.com/smartsdk/mongo-rs-controller-swarm/master/docker-compose.yml
-docker stack deploy -c docker-compose.yml ${STACK_NAME}
+curl https://raw.githubusercontent.com/smartsdk/mongo-rs-controller-swarm/master/docker-compose.yml -o mongo-docker-compose.yml
```

Comment:
this way you will every time retrieve the files. wouldn't be cleaner to have a separate "build" file?

Reply:
of course, if the files are not present we should call the build script then.

Reply:
correct, and that's it's intentional since it's only a few bytes (hence fast), has the benefit of "auto-update" when recipes are fixed.

Before implementing anything smarter here, we should discuss how we'll solve this for the whole repo. A more interesting option for example would be to get rid of the "deploy" shell scripts and have a single "update command" that pulls everything so that we can only focus on docker-stack deploy.